VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Public/GenerateCompletedCallbackProxy.h

```diff
@@ -32,15 +32,19 @@ class VITRUVIO_API UGenerateCompletedCallbackProxy final : public UBlueprintAsyn
 
 public:
 	DECLARE_DYNAMIC_MULTICAST_DELEGATE(FGenerateCompletedDelegate);
+	DECLARE_MULTICAST_DELEGATE(FGenerateCompletedDelegateCpp);
 	DECLARE_DYNAMIC_MULTICAST_DELEGATE(FOnAttributesEvaluatedDelegate);
+	DECLARE_MULTICAST_DELEGATE(FOnAttributesEvaluatedDelegateCpp);
 
 	/** Called after the attributes have been evaluated. Note that it is not guaranteed that this callback is ever called. */
 	UPROPERTY(BlueprintAssignable, meta = (DisplayName = "Attributes Evaluated"), Category = "Vitruvio")
 	FOnAttributesEvaluatedDelegate OnAttributesEvaluated;
+	FOnAttributesEvaluatedDelegateCpp OnAttributesEvaluatedCpp;

```

Comment:
We could probably use a more descriptive name here

Reply:
Ah yes typically we actually do the opposite. OnAttributesEvaluatedBlueprint and then using the DisplayName to override a nice name for the Blueprints. Will change that.